CHANGELOG.rst

```diff
@@ -1,5 +1,31 @@
 .. _changelog:
 
+0.32.0
+------
+
+Renku ``0.32.0`` introduces improvements in the KG services, thus enhancing KG overall performance. 
+
+User-Facing Changes
+~~~~~~~~~~~~~~~~~~~
+
+**✨ Improvements**
+
+- 🔁 **KG**: ``updatedAt`` renamed to ``dateModified`` on the Project Details API (`#1582 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1582>`_).
+- 📖 **KG**: Improved performance of the Project Status API (`#1554 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1554>`_).
+
+Internal Changes
+~~~~~~~~~~~~~~~~
+
+** Improvements**
+
+- **KG**: A new process to synchronize various project properties between GitLab and Triples Store (`#1569 <https://github.com/SwissDataScienceCenter/renku-graph/issues/1569>`_).
+- **KG**: A PostgreSQL DB is added for the triples-generator (`values changelog <https://github.com/SwissDataScienceCenter/renku/blob/master/helm-chart/values.yaml.changelog.md#upgrading-to-renku-0290>`_).
```

Comment:
afaik this has been there for a while?

Reply:
That's true. It should have been added to the `0.29.0` release notes. I'll add it retrospectively so anyone upgrading from an older version will be warned.